app/src/main/java/org/fossasia/openevent/app/common/utils/core/ErrorUtils.java

```diff
@@ -1,39 +1,49 @@
 package org.fossasia.openevent.app.common.utils.core;
 
+import org.json.JSONObject;
+
+import okhttp3.ResponseBody;
 import retrofit2.HttpException;
 
 public final class ErrorUtils {
 
-    public static final int BAD_REQUEST = 400;
-    public static final int UNAUTHORIZED = 401;
-    public static final int FORBIDDEN = 403;
-    public static final int NOT_FOUND = 404;
-    public static final int METHOD_NOT_ALLOWED = 405;
-    public static final int REQUEST_TIMEOUT = 408;
-
     private ErrorUtils() {
         // Never Called
     }
 
     public static String getMessage(Throwable throwable) {
```

Comment:
Nice job. Can you please add tests for these as well

Reply:
Apologies for furthur delay but I'm quite busy with my mid sems and proposal, will update it soon.